CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ All notable changes to this project will be documented in this file.
 
 ## [4.X.X](https://github.com/ReactiveX/RxSwift/releases/tag/4.X.X)
 
+* Xcode 9.3 / Swift 4.1 compatibility
```

Comment:
I would change this line to "Increase minimum requirement to Swift 4.1". (and we should also make sure that's updated in the README)